### PR TITLE
Configurable default values for the components

### PR DESCRIPTION
**Pull request by bas080**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export class Config {
+
+  configurations = {
+    range: 3,
+    limit: 30,
```
Comment (VMBindraban) [inline src/config.js]: Also add `criteria` (empty object).

File: src/config.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  }
+
+  configure(configs) {
+    Object.assign(this.configurations, configs);
+    return this;
```
Comment (VMBindraban) [inline src/config.js:12]: new line before `return`

**Comment by doktordirk** [comment]
can you rebase

**Comment by RWOverdijk** [comment]
@bas080 Is this something needed still?

**Pull request closed without merging**